tests/test_crew.py

```diff
@@ -1,19 +1,19 @@
 from datadog import statsd
 import logging
+import mock
 import os
 import pytest
 import sys
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'sqsworkers')))
-
-import crew
-
-class MsgProcessor():
-    def __init__(self):
-        logging.getLogger('default').info('msg processor instantiated')
+import traceback
 
-    def start(self):
-        logging.getLogger('default').info('processed')
+from helpers import aws_adapter_for_testing
+from helpers import BulkMsgProcessor
+from helpers import MockAWSAccount
```

Comment:
This import is unused; it should be removed

Reply:
removing this.